src/ibmca_pkey.c

```diff
@@ -722,3 +664,7 @@ void ibmca_ed448_destroy(void)
         ibmca_ed448_pmeth = NULL;
     }
 }
+
+#ifdef OPENSSL_VERSION_PREREQ
+/* copied since symbol not available */
+#endif
```

Comment:
what is this for ?

Reply:
Ups, left-over that I deleted but did not save the file...